plugins/woocommerce/includes/admin/class-wc-admin-settings.php

```diff
@@ -459,7 +459,7 @@ class="<?php echo esc_attr( $value['class'] ); ?>"
 								<fieldset>
 									<?php
 									if ( ! $show_desc_at_end ) {
-										echo wp_kses_post( $description );
+										echo $description; // WPCS: XSS ok.
```

Comment:
We should use `phpcs:ignore` here instead, and it's good to indicate why the rule is being ignored.

```suggestion
										echo $description; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- description is already sanitized in WC_Admin_Settings::get_field_description().
```